examples/energy_star.yaml

```diff
@@ -0,0 +1,43 @@
+defaults:
+  - backend: pytorch # default backend
+  - launcher: process # default launcher
+  - benchmark: energy_star # default benchmark
+  - experiment # inheriting experiment schema
+  - _self_ # for hydra 1.1 compatibility
+  - override hydra/job_logging: colorlog # colorful logging
+  - override hydra/hydra_logging: colorlog # colorful logging
+
+experiment_name: energy_star
+
+backend:
+  device: cpu
+  device_ids: 0
+  no_weights: true
+  model: gpt2
+  task: feature-extraction
```

Comment:
`text-generation` is a bit different as the model has a head on top of the backbone. I just started with `feature-extraction` because it was the easier way to start in terms of preprocessing. But we'll have another preprocessing method for `text-generation`!

Reply:
thanks ! make sense !